app/controllers/controller_with_authentication.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Provides authentication to derived controllers.
+# User is exposed via current_user method.
+class ControllerWithAuthentication < ApplicationController
+  rescue_from(
+    Authentication::JwtCodec::InvalidTokenError,
+    Http::BearerTokenFromHeaders::MissingToken,
+    Http::BearerTokenFromHeaders::UnsupportedType,
+    with: :unauthenticated
+  )
```

Comment:
This `rescue_from` does not work for some reason, I've open the issue #30 and asked about it on StackOverflow https://stackoverflow.com/questions/68912462/changing-render-key-from-plain-to-json-causes-server-error